geode-core/src/main/java/org/apache/geode/internal/cache/EvictionAttributesImpl.java

```diff
@@ -159,5 +163,73 @@ public boolean isLIFOMemory() {
     return this.algorithm == EvictionAlgorithm.LIFO_MEMORY;
   }
 
+  public static EvictionAttributesImpl fromConfig(
+      RegionAttributesType.EvictionAttributes configAttributes)
+      throws ClassCastException, InstantiationException,
+      IllegalAccessException {
+    EvictionAttributesImpl evictionAttributes = new EvictionAttributesImpl();
+    if (configAttributes == null) {
+      evictionAttributes.setAction(EvictionAction.NONE);
```

Comment:
I am wondering should we also have a similar concept in our config object to indicate NONE. If the update case, if user pass in null, that means I want to leave the existing configuration alone, if they pass in NONE, that means I want to wipe out the previous configuration.

Reply:
There's nothing in the XML POJO that indicates that, and we didn't want to touch it. But yes, that would be a better abstraction.